src/build_vehicle_gui.cpp

```diff
@@ -39,6 +41,60 @@
 
 #include "safeguards.h"
 
+struct CargoFilter
+{
```

Comment:
The opening brace of struct and class types should be on the same line as the type name. Only function definitions have the opening brace on a line by itself.